CI/travis/lib.sh

```diff
@@ -155,9 +155,9 @@ get_ldist() {
 }
 
 __brew_install_or_upgrade() {
-	brew install $1 || \
-		brew upgrade $1 || \
-		brew ls --versions $1
+	brew install "$1" || \
```

Comment:
IIRC: I did this on purpose [not adding quotes];
but it may be fine to add it;
not sure

Reply:
It doesn't change anything - and does protect things...